frigate/util.py

```diff
@@ -429,6 +430,39 @@ def copy_yuv_to_position(
     source_frame=None,
     source_channel_dim=None,
 ):
+    def crop_clear(frame, position, fill_value):
+        frame[position[1] : position[3], position[0] : position[2]] = fill_value
+
+    def assign_resized_frame(
+        destination_slice, source_slice, resize_dim, interpolation
+    ):
+        source_img = Image.fromarray(source_slice)
```

Comment:
I think its likely the performance gain here has nothing to do with using PIL. Using opencv's `cv2.INTER_NEAREST` probably gets the same performance gain without casting back and forth from a numpy array.

Reply:
some optimisations about casting numpy array made in a2b30d0

Reply:
I would still prefer to use opencv's resize unless there is a good reason to switch since we use that in almost every other part of the code base.

Reply:
but it give performance improvement... Especially with enhanced multiprocessing (#6986 and #6936)

Reply:
can you repeat this test with #7053 merged in? This can significantly affect the result.And will be wonderful to see py-spy output

Reply:
I will try it, but if that PR made a difference it would lower the CPU usage for all cases not just the PIL case

Reply:
of course. This PR provides 2 optimisations. Maybe, in some cases, an error in the second optimization neutralizes the benefits of the first one. need to check